ui/shared/components/panel/acmg/AcmgRuleSpecification.jsx

```diff
@@ -89,16 +94,17 @@ const tableListItemsForCriteria = (items, criteria) => {
 
 const tableRowWithTableCells = (items, description, cellsWithList = false, color = 'blue') => {
   const row = (
-    <Table.Row>
+    <Table.Row key={description}>
       <Table.Cell textAlign="center">{description}</Table.Cell>
-      {items.map((item) => {
+      {items.map((item, key) => {
```

Comment:
It would be better to add a key field in each item.

Reply:
agreed, but see my comment above